src/main/js/controls/Line/helpers/helpers.js

```diff
@@ -511,16 +511,17 @@ const drawDataPoints = (scale, config, pointGroupPath) => {
 };
 /**
  * Handler for Request animation frame, executes on resize.
- *  * Order of execution
- *      * Shows/hides the regions
+ * * Order of execution
+ * * Shows/hides the regions
```

Comment:
looks like we need gap between two stars in here to state that order of execution is show/hide the regions.